var/spack/repos/builtin/packages/py-onnxruntime/package.py

```diff
@@ -22,6 +22,7 @@ class PyOnnxruntime(CMakePackage, PythonExtension):
 
     license("MIT")
 
+    version("1.17.3", tag="v1.17.3", commit="56b660f36940a919295e6f1e18ad3a9a93a10bf7")
```

Comment:
Curious:  Why not use the more secure `sha256`?

```suggestion
    version("1.17.3", sha256="4112f6552c17216311be013b9a2621a3c1a17ed9e1268a1142599b6a494517c2")
```

Reply:
I'm just reusing what was there , i.e. git tag based definition by commit.

The source archives contain the submodules as well, which we don't want (i.e. we don't want to pull in an entire copy of onnx as a submodule as well since we want to use the spack-installed version).